src/lib/rest/rest.cpp

```diff
@@ -413,17 +414,26 @@ static int outFormatCheck(ConnectionInfo* ciP)
 
 /* ****************************************************************************
 *
-* servicePathCheck - 
+* servicePathCheck - check vector of service paths
 *
-* Not static just to let unit tests call this function
+* This function is called for ALL requests, when aservice-path URI-parameter is found.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/a91b5469331a9fa51ea89e994fc815f9b7b24072